.circleci/config.yml

```diff
@@ -22,8 +29,38 @@ jobs:
       - run:
           name: Run lint
           command: make lint
+      - run: scripts/build-glean-metadata.py
+      - run: mkdir /tmp/workspace-data
+      - run: cp -r public/data/* /tmp/workspace-data
+      - persist_to_workspace:
+          root: /tmp/workspace-data
+          paths:
+            - ./*
+  upload:
+    # this job uploads the built artifacts to the GCS bucket
+    # used to serve https://dictionary.protosaur.dev/
+    docker:
+      - image: google/cloud-sdk
+    working_directory: /tmp
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - attach_workspace:
+          at: /tmp/workspace-data
+      - run: cp -r /tmp/workspace-data/* /tmp/workspace/public/data
+      - gcs/gcs-rsync:
+          bucket: glean-dictionary-dev
+          source: /tmp/workspace/public
+          options: -r
```

Comment:
Is `-d` appropriate here when files are removed?

Reply:
I think so, can add this.